dev-utils/test-config.js

```diff
@@ -119,10 +119,10 @@ function getBrowserList() {
       platformName: 'android'
     },
     {
-      appiumVersion: '1.9.1',
+      appiumVersion: '1.13.0',
       deviceName: 'iPhone Simulator',
       deviceOrientation: 'portrait',
-      platformVersion: '11.1',
+      platformVersion: '12.2',
```

Comment:
Why do we need to drop support for iOS 11?

Reply:
We are not dropping, 12.2 has the major share so i just updated it since we already test for safari 9.1 in mac. least + current.